README.md

```diff
@@ -2,26 +2,6 @@
 
 [![Build status](https://badge.buildkite.com/f21b82191811f668ef6fe24f6151058a84fa2c645cfa8810d0.svg?branch=master)](https://buildkite.com/uberopensource/fusion-core)
 
-### Guides
-
-* [What is Fusion.js](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/what-is-fusion.md)
-* [Getting started](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/getting-started.md)
-* [Framework comparison](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/framework-comparison.md)
-
-### Core concepts
-
-* [Universal code](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/universal-code.md)
-* [Creating a plugin](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/creating-a-plugin.md)
-  * [Dependencies](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/dependencies.md)
-  * [Creating endpoints](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/creating-endpoints.md)
-  * [Creating providers](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/creating-providers.md)
-  * [Modifying the HTML template](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/modifying-html-template.md)
-  * [Working with secrets](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/working-with-secrets.md)
-
----
-
-## fusion-core
```

Comment:
Should probably keep this line

Reply:
We have the same title on line 1, `# fusion-core`. It looks good I promise :)